pages/implementation/inspector/sdk/dart.mdx

```diff
@@ -0,0 +1,99 @@
+---
+layout: 'guide'
+title: 'Avo Inspector Flutted and Dart SDK'
+---
+
+import Link from '../../../../components/Link';
+
+[Package on Pub.dev](https://pub.dev/packages/avo_inspector)
+
+## <a name="quick-start"></a>Quick Start Guide
+
+Find the Quick Start Guide in our <a target="_blank" rel="noopener noreferrer" href="https://github.com/avohq/flutter-avo-inspector">GitHub repo.</a>
+
+## <a name="installation"></a> Installation
+
+With Flutter:
+
+$ flutter pub add avo_inspector
+
+This will add a line like this to your package's pubspec.yaml (and run an implicit flutter pub get):
+
+dependencies:
+  avo_inspector: ^0.9.9
+
+## <a name="import"></a>Import
+
+Now in your Dart code, you can use:
+
+```dart
+import 'package:avo_inspector/avo_inspector.dart';
+```
+
+## <a name="initialization"></a>Initialization
+
+Obtain the API key in the Inspector tab in your <a target="_blank" rel="noopener noreferrer" href="https://www.avo.app/welcome">Avo.app workspace</a>.
```

Comment:
Nice to have the link here. It would be super nice if we could deep-link without specifying a workspace.